### PR TITLE
Bs/650974736/free draw controls

### PR DESCRIPTION
**Pull request by bethstevic**
[Card](https://rentpath.leankit.com/card/650974736)

- Display drawing controls
- Display 'within zoom level' label
- Display 'outside of zoom level' label
- Displays or not based on zoom lev

### CODE REVIEW

File: packages/react-ui-ag/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+## [6.5.1](https://github.com/rentpath/react-ui/compare/@rentpath/react-ui-ag@6.5.0...@rentpath/react-ui-ag@6.5.1) (2018-06-21)
+
+
+### Bug Fixes
+
+* **@rentpath/react-ui-ag:** fixes issue where mobile listing was not passing renderCustomControls th ([1aea52f](https://github.com/rentpath/react-ui/commit/1aea52f))
+
+
+
+
```
Comment (bethstevic) [inline packages/react-ui-ag/CHANGELOG.md]: This is showing up because I rebased with the master branch

File: packages/react-ui-ag/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-ag",
-  "version": "6.5.0",
+  "version": "6.5.1",
```
Comment (bethstevic) [inline packages/react-ui-ag/package.json]: This is showing up because I rebased with the master branch

File: stories/theme/core/Gmap.css [hunk 3]
```diff
@@ -24,7 +41,74 @@
...
+.Gmap_FreeDrawBanner-draw  .Text,
+.Gmap_FreeDrawBanner-zoomIn .Text {
+  max-width: 50%;
+  font-size: 12px;
+}
+
+.Gmap_FreeDrawBanner-zoomIn {
+  -moz-animation: fadeOut 3s ease-in 3s forwards; /* Firefox */
+  -webkit-animation: fadeOut 3s ease-in 3s forwards; /* Safari and Chrome */
+  -o-animation: fadeOut 3s ease-in 3s forwards; /* Opera */
```
Comment (tadjohnston) [inline stories/theme/core/Gmap.css]: `autoprefixer` is turned on so you shouldn't need any of these.

File: stories/core/ExampleComponents/CustomBannerExample.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(/^CustomBannerExample/, { pure: true })
+
```
[the remark this answers is not in the document]
Comment (bethstevic) [reply stories/core/ExampleComponents/CustomBannerExample.js]: There shouldn't be a space after `@themed` - it is a function call

File: packages/react-ui-core/src/Gmap/utils/__tests__/mapEventHelpers-test.js [hunk 5]
```diff
@@ -58,6 +58,32 @@ describe('mapEventHelpers', () => {
     expect(actual).toEqual(expected)
   })
 
+  it('sets up events with addListenerOnce if the once argument is set to true', () => {
```
Comment (tadjohnston) [inline packages/react-ui-core/src/Gmap/utils/__tests__/mapEventHelpers-test.js]: Looks like you rebases this whole section wrong: https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Gmap/utils/__tests__/mapEventHelpers-test.js

File: stories/core/ExampleComponents/CustomBannerExample.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (orionstein) [reply stories/core/ExampleComponents/CustomBannerExample.js]: We should have no space after decorators

**Pull request merged**